enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -1124,8 +1132,18 @@ func (e *partitionEvictor) deleteFile(sample *evictionPoolEntry) error {
 		return err
 	}
 	e.atimes.Delete(xxhash.Sum64(sample.fileMetadataKey))
-	if err := disk.DeleteFile(context.TODO(), sample.filePath); err != nil {
-		return err
+
+	md := sample.fileMetadata.GetStorageMetadata()
+	switch {
+	case md.GetFileMetadata() != nil:
+		fp := filestore.FilePath(e.blobDir, md.GetFileMetadata())
+		if err := disk.DeleteFile(context.TODO(), fp); err != nil {
```

Comment:
You want to use background context? Or leave as TODO?

Reply:
same same (i copied this from disk_cache.go)